test/data/conformance/no-financial.xml

```diff
@@ -96,6 +63,12 @@
           <eeSummary:radiationExposureInd>true</eeSummary:radiationExposureInd>
         </eeSummary:specialFactors>
       </eeSummary:enrollmentDeterminationInfo>
+      <eeSummary:financialsInfo xmlns:eeSummary="http://jaxws.webservices.esr.med.va.gov/schemas">
```

Comment:
Yeah I was very confused about how to fix the conformance tests, this might be included because of the logic changes you identified earlier, so once I make those this should go away, I just copied it in based on wha the failing test expected to see.